### PR TITLE
Implement settings API on admin settings payment-gateway

### LINKED ISSUE #529: Implement Settings API to /admin/settings/payment-gateway
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

Work to be done:
- Implement the settings API to /admin/settings/payment-gateway

### PR DESCRIPTION
**Pull request by CosmicCoder96**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Implement settings API on admin settings payment-gateway

Fixes #529

### CODE REVIEW

**Comment by CosmicCoder96** [comment]
Deployment link : https://cc96-opev-frontend.herokuapp.com/admin/settings/payment-gateway

**Comment by CosmicCoder96** [comment]
@niranjan94 There are no fields on the API for the checkboxes to enable/disable paypal and stripe, should I remove the checkboxes?

**Comment by niranjan94** [comment]
> There are no fields on the API for the checkboxes to enable/disable paypal and stripe, should I remove the checkboxes?

If `paypalSandboxUsername` or `paypalLiveUsername` is not null, then that is enabled. And if disabled, make sure all paypal* fields are made null
If `stripeClientId` is not null, then that is enabled. And if disabled, make sure all stripe* fields are made null.

**Comment by CosmicCoder96** [comment]
@niranjan94 Please review

File: app/components/forms/admin/settings/payment-gateway-form.js [hunk 1]
```diff
@@ -102,10 +102,40 @@ export default Component.extend(FormMixin, {
       }
     };
   },
+  isCheckedStripe: computed(function() {
+    if (this.get('settings').get('stripeClientId')) {
```
Comment (niranjan94) [inline app/components/forms/admin/settings/payment-gateway-form.js]: Why two gets? Use dot notation

File: app/components/forms/admin/settings/payment-gateway-form.js [hunk 2]
```diff
@@ -102,10 +102,40 @@ export default Component.extend(FormMixin, {
...
+      return true;
+    }
+  }),
+  isCheckedPaypal: computed(function() {
+    if (this.get('settings').get('paypalSandboxUsername') || this.get('settings').get('paypalLiveUsername')) {
+      return true;
```
Comment (niranjan94) [inline app/components/forms/admin/settings/payment-gateway-form.js]: And instead of this you can directly return the condition

**Comment by CosmicCoder96** [comment]
@niranjan94 Done 👍

**Comment by CosmicCoder96** [comment]
@niranjan94 Thanks! I have made those changes.

**Pull request merged**